tests/parser/exceptions/test_constancy_exception.py

```diff
@@ -70,7 +70,7 @@ def b():
 f:int128
 
 @public
-def a(x:num):
+def a (x:int128):
```

Comment:
Why is there a space between `a` and the parens?

Reply:
must've been a typo that slipped in, wil fix.